cli/azd/pkg/project/scaffold_gen.go

```diff
@@ -299,49 +333,15 @@ func mapContainerApp(res *ResourceConfig, svcSpec *scaffold.ServiceSpec, infraSp
 		// Here, DB_HOST is not a secret, but DB_SECRET is. And yet, DB_HOST will be marked as a secret.
 		// This is a limitation of the current implementation, but it's safer to mark both as secrets above.
 		evaluatedValue := genBicepParamsFromEnvSubst(value, isSecret, infraSpec)
-		svcSpec.Env[envVar.Name] = evaluatedValue
+		serviceSpec.Env[envVar.Name] = evaluatedValue
 	}
 
 	port := props.Port
 	if port < 1 || port > 65535 {
-		return fmt.Errorf("port value %d for host %s must be between 1 and 65535", port, res.Name)
-	}
-
-	svcSpec.Port = port
-	return nil
-}
-
-func mapHostUses(
-	res *ResourceConfig,
-	svcSpec *scaffold.ServiceSpec,
-	backendMapping map[string]string,
-	prj *ProjectConfig) error {
-	for _, use := range res.Uses {
-		useRes, ok := prj.Resources[use]
-		if !ok {
-			return fmt.Errorf("resource %s uses %s, which does not exist", res.Name, use)
-		}
-
-		switch useRes.Type {
-		case ResourceTypeDbMongo:
-			svcSpec.DbCosmosMongo = &scaffold.DatabaseReference{DatabaseName: useRes.Name}
-		case ResourceTypeDbPostgres:
-			svcSpec.DbPostgres = &scaffold.DatabaseReference{DatabaseName: useRes.Name}
-		case ResourceTypeDbRedis:
```

Comment:
The original code handles the bindings here, too. Here respects `uses`, but not include all bindings. For example: Its doesn't contain mysql / service bus / event hubs here.